static/app/icons/iconUnlock.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import SvgIcon from './svgIcon';
+
+type Props = React.ComponentProps<typeof SvgIcon>;
+
+const IconUnlock = React.forwardRef(function IconUnlock(
+  props: Props,
+  ref: React.Ref<SVGSVGElement>
+) {
+  return (
+    <SvgIcon {...props} ref={ref}>
+      <g clipPath="url(#a)">
```

Comment:
@vuluongj20 I double-checked this icon and the paths are bigger than the dimensions of the SVG itself, probably why you have this `clipPath` prop.

Check it out.

<img width="1152" alt="Screen Shot 2021-12-03 at 16 10 38" src="https://user-images.githubusercontent.com/6508377/144625596-3a14f79b-f05d-48b7-b9d6-26adc34c73bf.png">